src/main/scala/Day07Copy.scala

```diff
@@ -0,0 +1,74 @@
+import scala.io.*
+
+object Day07Copy extends App:
+
+  private val day: String =
+    this.getClass.getName.drop(3).init
+
+  val start1: Long = System.currentTimeMillis
+  val start2: Long = System.currentTimeMillis
```

Comment:
note `val`s are initialised in order when the object is initialised; ie. start2 has the same millis as start2